### DISCUSSION TITLE
How to use the crepe themes or create own themes and use them?

### DISCUSSION THREAD

**Discussion by Koffer0-0** [Milkdown/milkdown#1540] [Q&A]
I want to change the default theme 
```
import "@milkdown/crepe/theme/common/style.css";
import "@milkdown/crepe/theme/frame.css";
```

to this
```
import "@milkdown/crepe/theme/nord/style.css";
import "@milkdown/crepe/theme/frame.css";
```
It gives me error 

Missing "./theme/nord/style.css" specifier in "@milkdown/crepe" package

IDK But the [documentation](https://milkdown.dev/docs/guide/using-crepe#theme) is not clear.

**Comment by greyovo** [comment]
https://milkdown.dev/docs/guide/styling
